File: app/src/main/java/com/loma/project/model/Project.java
package com.loma.project.model;

import android.content.Intent;
import android.util.Log;
import android.view.View;

import com.loma.project.ProjectDaterailActivity;

public class Project {
    public String date;
    public String  vesion;
    public String  endDate;
    public boolean android;
    public String  icon;
    public String  description;
    public String  testURL;
    public String  type;
    public boolean ios;
    public String  devURL;
    public boolean pc;
    public String  name;
    public int     progress;
    public int     id;
    public String  src_vcs;

    public void  read(View view){
       view.getContext().startActivity(new Intent(view.getContext(),ProjectDaterailActivity.class));
    }
}

File: app/src/main/java/db/Project.kt
package db

import io.objectbox.annotation.Entity
import io.objectbox.annotation.Id

@Entity
class Project {
    @Id(assignable = true)
    var id: Long = 0
    var date: String? = null
    var vesion: String? = null
    var endDate: String? = null
    var android: Boolean = false
    var icon: String? = null
    var description: String? = null
    var testURL: String? = null
    var type: String? = null
    var ios: Boolean = false
    var devURL: String? = null
    var pc: Boolean = false
    var name: String? = null
    var progress: Int = 0


    var src_vcs: String? = null
}
